### PR TITLE
Add initial reading position component docs

### PR DESCRIPTION
**Pull request by godleman**
### Description of the Change

Add initial documentation for the reading position component - based on the existing docs format.

### Alternate Designs

N/A

### Benefits

Component library update

### Possible Drawbacks

Will need review to check for doc errors.

### Verification Process

What process did you follow to verify that your change has the desired effects?

I did not have a local build of the component library so unable to verify the following:
- How did you verify that all new functionality works as expected? No 
- How did you verify that all changed functionality works as expected? No
- How did you verify that the change has not introduced any regressions? No

### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.

### Applicable Issues

The main issue is that I do not a have baseline local component library build to check the changes on the frontend.

### CODE REVIEW

**Comment by timwright12** [comment]
Holding this back since we don't have the component published yet.

File: _component/reading-position/example.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		<title>10up Component Library | Reading Position Indicator</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<!-- It is not recommended to use the JSDelivr CDN for production. -->
+		<!-- Instead please install the components with npm or by including the files directly in your project -->
+		<style>body {margin: 0;padding: 0;}</style>
+		<link href="https://cdn.jsdelivr.net/npm/@10up/component-accordion@latest/dist/reading-position.css" rel="stylesheet">
+	</head>
+	<body>
+		<div class="reading-position-example">
```
Comment (timwright12) [inline _component/reading-position/example.html:13]: @cldhmmr can we use normal Lorem Ipsum for this?

**Comment by jeffpaul** [comment]
@timwright12 has the respective component been published such that this PR can/should continue through review?

**Comment by timwright12** [comment]
This got merged somewhere else, closing out.

**Pull request closed without merging**